projects/lf-documentation/src/app/getting-started/getting-started.component.html

```diff
@@ -17,14 +17,15 @@
   </ul>
 </div>
 <h2 id="cdn-usage">Using Via CDN</h2>
+<p>Laserfiche UI-components can be imported via any CDN. Below, we use jsdelivr.</p>
```

Comment:
Laserfiche UI Components can be imported using any CDN hosting the npm package, including jsdelivr, unpkg, and lfxstatic. Below, we use jsdelivr.